tests/ha/single_pod_failure/test_data_pod_failure.py

```diff
@@ -2859,8 +2859,11 @@ def test_object_crud_pod_failure(self):
 
         num_replica = 0
         LOGGER.info("Get pod to be deleted")
-        pod_list = self.node_master_list[0].get_all_pods(pod_prefix=const.POD_NAME_PREFIX)
-        delete_pod = pod_list[-1]
+        sts_dict = self.node_master_list[0].get_sts_pods(pod_prefix=const.POD_NAME_PREFIX)
```

Comment:
are we not adding this setup_method as we did in server failure file?

Reply:
We will add this to setup_method after doing changes in all the tests. For now I will keep this as is.